### PR TITLE
Wireless charger

### LINKED ISSUE #201: Feature Request: Check if device supports Wireless Charger
This is taken from apple website, 

Screen Shot 2019-09-11 at 17 30 15

### PR DESCRIPTION
**Pull request by gianpispi**
Added wireless charging capability variable

Closes #201

### CODE REVIEW

**Comment by gianpispi** [comment]
Sorry @Zandor300, what do you mean by that?

**Comment by Zandor300** [comment]
@gianpispi You added a property to the initializer of the gyb Device class. Now you also need to add this boolean for every device that is in there so that the `allDevicesWithWirelessChargingSupport` and `supportsWirelessCharging` functions will be auto-generated from that.

**Comment by gianpispi** [comment]
Okay got it! Now I think it's all right. Thanks for helping ;)

File: Source/Device.swift.gyb [hunk 1]
```diff
@@ -35,65 +35,65 @@ class Device:
...
-            Device("iPadPro11Inch",  "Device is an [iPad Pro 11-inch](https://support.apple.com/kb/SP784)",                    "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP784/ipad-pro-11-2018_2x.png",                ["iPad8,1", "iPad8,2", "iPad8,3", "iPad8,4"], 11.0, (139, 199), "iPad Pro (11-inch)", 264, False, False, True, False, False, True, False, True, 2, False, 1),
-            Device("iPadPro12Inch3", "Device is an [iPad Pro 12.9-inch (3rd generation)](https://support.apple.com/kb/SP785)", "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP785/ipad-pro-12-2018_2x.png",                ["iPad8,5", "iPad8,6", "iPad8,7", "iPad8,8"], 12.9, (512, 683), "iPad Pro (12.9-inch) (3rd generation)", 264, False, False, True, False, False, True, False, True, 2, False, 1)
+            Device("iPad2",          "Device is an [iPad 2](https://support.apple.com/kb/SP622)",                              "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP622/SP622_01-ipad2-mul.png",                 ["iPad2,1", "iPad2,2", "iPad2,3", "iPad2,4"], 9.7,  (3, 4),     "iPad 2", 132, False, False, False, False, False, False, False, False,False, 0, False, 1),
```
Comment (Zandor300) [inline Source/Device.swift.gyb]: Please add a space between the False,False

File: Source/Device.swift.gyb [hunk 2]
```diff
@@ -35,65 +35,65 @@ class Device:
...
+            Device("iPad3",          "Device is an [iPad (3rd generation)](https://support.apple.com/kb/SP647)",               "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP662/sp662_ipad-4th-gen_color.jpg",           ["iPad3,1", "iPad3,2", "iPad3,3"],            9.7,  (3, 4),     "iPad (3rd generation)", 264, False, False, False, False, False, False, False, False,False, 0, False, 1),
```
Comment (Zandor300) [inline Source/Device.swift.gyb]: Please add a space here too.

File: Source/Device.generated.swift [hunk 3]
```diff
@@ -721,7 +728,7 @@ public enum Device {
     public var has3dTouchSupport: Bool {
       return isOneOf(Device.allDevicesWith3dTouchSupport)
     }
-  
+
```
Comment (Zandor300) [inline Source/Device.generated.swift]: Please remove the trailing whitespace. You might also have to change this in the gyb file.

File: Source/Device.swift.gyb [hunk 4]
```diff
@@ -35,65 +35,65 @@ class Device:
...
+            Device("iPadAir3",       "Device is an [iPad Air (3rd generation)](https://support.apple.com/kb/SP787)",           "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP787/ipad-air-2019.jpg",                      ["iPad11,3", "iPad11,4"],                     10.5, (3, 4),     "iPad Air (3rd generation)", 264, False, False, False, False, True, False, False, False, False, 1, False, 1),
+            Device("iPad7",          "Device is an [iPad (7th generation)](https://support.apple.com/kb/SP807)",               "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP807/sp807-ipad-7th-gen_2x.png",              ["iPad7,11", "iPad7,12"],                     10.2, (3, 4),     "iPad (7th generation)", 264, False, False, False, False, True, False, False, False, False, 1, False, 1),
```
Comment (Zandor300) [inline Source/Device.swift.gyb]: There already is an PR for this change. #206

Comment (gianpispi) [reply Source/Device.swift.gyb]: Oh okay, didn't see it. But it misses the wireless charging parameter.

Comment (Zandor300) [reply Source/Device.swift.gyb]: That PR is now merged, so if you rebase your PR with master, you can add the value to it.

File: Source/Device.generated.swift (hunk 3 above)
Comment (Zandor300) [reply Source/Device.generated.swift]: This one still stands

Comment (gianpispi) [reply Source/Device.generated.swift]: That code is outdated. If you look at the code itself there is no space.

File: Source/Device.swift.gyb (hunk 4 above)
Comment (gianpispi) [reply Source/Device.swift.gyb]: Did it, and it should be all fine now.

**Pull request merged**